bedrock/firefox/templates/firefox/features/send-tabs.html

```diff
@@ -0,0 +1,151 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% add_lang_files "firefox/shared" "firefox/sync" %}
+
+{% extends "firefox/features/base.html" %}
+
+{% block page_title %}
+  {{ _('Take your browser tabs to go | Firefox') }}
+{% endblock %}
+
+{% block page_desc %}
+  {{ _('Send your desktop browser tabs to phones, tablets and more with your Firefox Account.') }}
+{% endblock %}
+
+{% block page_css %}
+  {{ super() }}
+  {% stylesheet 'firefox-features-hub-detail' %}
+  {% stylesheet 'firefox-features-sync' %}
+{% endblock %}
+
+{% block body_id %}firefox-features-send-tabs{% endblock %}
+
+{% block body_class %}
+  {{ super() }}
+  state-default
+{% endblock %}
+
+{% block features_header_image %}
+  {{ high_res_img('firefox/features/send-tabs-hero.png', {'class': 'hero-image', 'alt': '', 'width': '700', 'height': '390'}) }}
+{% endblock %}
+
+{% block features_header_content %}
+  <h1><span>{{ _('Take your tabs to go') }}</span></h1>
+  <p class="tagline">{{ _('Share your open tabs seamlessly across mobile, desktop or tablet with Firefox.') }}</p>
+{% endblock %}
+
+{% block features_list %}
+<section>
+  <div class="content">
+    <h2 class="section-title"><span>{{ _('Send tabs') }}</span></h2>
+
+    <div id="sync-intro-copy-cta-wrapper">
+      <div id="sync-intro-copy">
+        {# Warning Text #}
+        <div class="show-fx-30-older warning">
+          <p>
+            <span>{{ _('It looks like you’re running an older version of Firefox.') }}</span>
+          </p>
+        </div>
+
+        <div class="show-fx-31-signed-in">
+          <strong>{{ _('Logged in and connected!') }}</strong>
+          <p>
+            {{ _('You’re all set to start sending tabs.') }}
```

Comment:
Sigh, yes. This is what happens when you skip a UI/UX phase and just go with "fork the Sync page and update the copy". Hopefully we get priority to do a phase 2.